src/app/core/config/config-fix.json

```diff
@@ -503,5 +503,30 @@
                 }
             ]
         }
+    },
+
+    "entity:Child": {
+        "attributes": [
+            {"name": "gender", "schema": { "dataType": "string" } },
```

Comment:
gender, mothertongue and religion are still defined in the class, also. Should we define every field in the config - or remove every field that is already defined in the class?

Reply:
I removed the duplicates from the config. We should soon clean up all helgo dependent attributes from the entities and tests, so we are not stuck with them on future deployments